doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,131 @@ Topics include:
+
+3. **Logs** are a record messages generated by software for most devices (compute and network) and virtual
+applications and transported over SYSLOG and tend to come in high volumes.
+
+4. **Traces** are end-to-end signalling messages (events) created to fullfil execution of requests on
+the distributed system services. OTHER WORDS: Traces are all action points executed in
+order to provide response to the request set to the distributed system service. Even the call
+can be thought of as a request which starts by INVITE message of the SIP protocol.
+
+<a name="9.6.1.4"></a>
+#### 9.6.1.4. Where to send the data
+
+If the observability data have to be sent from their sources (or producers) to specific destinations (or consumers), then this creates high degree of dependency between producers and consumers, and is extremely prone to errors, especially in case of configuration changes. Ideally, the data producers must not be impacted with any change in the data consumers and vice versa.
+This is achieved by decoupling data producers from data consumers through the use of Brokers. The Producers always send their data to the same endpoint - the Broker. While the Consumers register with the Broker for data that is of interest to them and always receive their data from the Broker.
+
+<a name="9.6.1.5"></a>
+#### 9.6.1.5. Which protocol, interface, and format to use
+
+While protocols and interfaces are dictated by the selection of the message broker (common data bus) system, data format is usually customizable according to the needs of users. The concept of Schema Registry mechanism, well known in the world of big data, is helpful here to make sure that message structures and formats are consistently used.
+
+<a name="9.6.2"></a>
+### 9.6.2. The Architecture
+
+In geographically dispersed large cloud deployments, a given telco cloud may have several cloud infrastructure components as well a large set of virtualized workloads (VNF/CNFs). It is important to monitor all of these workloads and infrastructure components. Furthermore, it is even more important to be able to correlate between the metrics provided by these entities to determine the performance and/or issues in such deployments. 
+
+The cloud deployment tends to shrink and expand based upon the customer demand. Therefore, an architecture is required that can scale on demand and does not force a strong tie between various entities. This means, the workloads and cloud infrastructure components that provide telemetry/performance metrics must not be burdened to discover each other. The capacity (e.g. speed, storage) of one component must not force overrun or underrun situations that would cause critical data to be lost or delayed to a point to render it useless. 
```

Comment:
```suggestion
The cloud deployment tends to shrink and expand based upon the customer demand. Therefore, an architecture is required that can scale on demand and does not force a strong tie between various entities. This means, the workloads and cloud infrastructure components that provide telemetry and performance metrics must not be burdened to discover each other. The capacity (e.g. speed, storage) of one component must not force overrun or underrun situations that would cause critical data to be lost or delayed to a point to render them useless. 
```